crates/ruff/src/rules/pydocstyle/rules/sections.rs

```diff
@@ -863,13 +866,13 @@ fn missing_args(checker: &mut Checker, docstring: &Docstring, docstrings_args: &
 }
 
 // See: `GOOGLE_ARGS_REGEX` in `pydocstyle/checker.py`.
-static GOOGLE_ARGS_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r"^\s*(\*?\*?\w+)\s*(\(.*?\))?\s*:(\r\n|\n)?\s*.+").unwrap());
+static GOOGLE_ARGS_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(r"^\s*(?P<arg_name>\*?\*?\w+)\s*(\(.*?\))?\s*:(\r\n|\n)?\s*.+").unwrap()
```

Comment:
Is this unchanged, apart from the use of the named parameter instead of the capture group index?

Reply:
Yup!

Reply:
If you'd prefer to keep it unnamed that's fine too - I just prefer using named groups if I need to pull one out.